add to readme for testing on code (#1207)
guardian/commercial: README.md

When your PR is merged, changeset will analyse the changes and create a PR to release the new version.

> For an efficient and secure process, a script is available to update the version of commercial lib in Frontend to your specified version. You can find the script [here](./bump_commercial.sh).

Reviewer: I think this should be in its own section and made a bit more obvious that the command is to be run in consumers i.e. frontend/DCR?

> For an efficient and secure process, a script is available to update the version of commercial lib in Frontend to your specified version. You can find the script [here](https://gist.github.com/chrislomaxjones/2663b9bcafd0ad61857b9cd6191c1a78).

Reviewer 2: I would recommend adding this script to the repo and linking to it here, rather than linking to Chris' gist.

When your PR is merged, changeset will analyse the changes and create a PR to release the new version.

> ### Running script to automatically produce a PR in Frontend

Reviewer 3: Minor opinion / non-blocking

```suggestion
### Bumping @guardian/commercial in Frontend
```

> For an efficient and secure process, a script is available to update the version of commercial lib in Frontend to your specified version. You can find the script [here](./bump_commercial.sh).

Reviewer 3: Minor opinion / non blocking

```suggestion
Run [this script](./scripts/bump_commercial.sh) to raise a PR that bumps `@guardian/commercial` in Frontend to the latest version.
```

> Save the script to your `\code` directory for easy execution from the terminal.

Reviewer 3: Can probably delete this line?

Execute the script as follows:

```bash

> ./bump_commercial.sh [VERSION_NUMBER]

Reviewer 3: ```suggestion
./scripts/bump_commercial.sh [VERSION_NUMBER]
```

```

Eg
```bash

> ./bump_commercial.sh 11.11.1

Reviewer 3: ```suggestion
./scripts/bump_commercial.sh 11.11.1
```
